### ISSUE TITLE
Integrate page object support

### ISSUE THREAD

**Issue by StephenDavidson** [electron/spectron#68]
It seems that spectron does not handle page objects implicitly, so implementing like the example below will result in a browser is defined error.

Docs:
http://webdriver.io/guide/testrunner/pageobjects.html

Example from webdriver io:
https://github.com/webdriverio/webdriverio/tree/master/examples/pageobject

**Comment by kevinsawicki** [comment]
> so implementing like the example below will result in a browser is defined error.

Have you tried replacing the `browser` calls with `app.client` calls instead? I think that should work.

**Comment by StephenDavidson** [comment]
app.client throws an ```app is not defined``` error. Here is a pr using pageobjects if you want to try out and double check. https://github.com/StephenDavidson/electron-spectron-example/pull/4

**Comment by kevinsawicki** [comment]
> app.client throws an app is not defined error. 

Yeah, `app` is not a global by default, you should pass in the value from your spec or set it as a global.

I commented on your pull request.

**Comment by StephenDavidson** [comment]
Works. The only issue I run into is that webdriverio implementation assumes sync tests, where it looks like spectron always runs async. Is there any option to force sync?

**Comment by christian-bromann** [comment]
@StephenDavidson you can just use async/await to build the same PO as WebdriverIO in sync mode.
